### ISSUE TITLE
single use to capturer code

### ISSUE THREAD

**Issue by zhihanhe** [perchco/perchrtc#33]
hi,ceaglest.Thank you so much for your code.It's help me a lot.

I have success use Renderers to our code.

But It's hard to move capturer code in our code. We want to filter the video .
the reason is the head file:
# include "talk/media/base/videocapturer.h"

in PerchRTC/CaptureKit/PHVideoCaptureBridge.h

videocapturer.h include other files.and the file location have changed to other folder.I've tried to move every file to project.But it's hard to find them.

so Is there a easy way to move your code in our code?

**Comment by difujia** [comment]
I guess you have a "header search path" problem. Try to add the project root (with recursive enabled) to header search path in the build settings.

**Comment by zhihanhe** [comment]
hi,difujia.thank you for your answer!It's really fix my problem.
But the webrtc api is changed...
some headers are deleted and api are change...so bad.

#include "webrtc/media/devices/devicemanager.h" is missing... ;(

thank you any way...

**Comment by ceaglest** [comment]
@zhihanhe Yes, the WebRTC APIs have changed a little since chrome 45 in this area. For one, many of the basic data types (int32, etc) have been redefined. If you are trying to build directly on top of WebRTC you will need to provide the headers needed for video capture, and build the capturer without RTTI.

**Comment by zhihanhe** [comment]
@ceaglest thank you for your answer.I find a thing I can not fixed.
Because of many api changed, I just extends RTCAVFoundationVideoSource class.
But It's refer a header file:
scoped_ptr.h
#include <algorithm> 

build failed due to "'algorithm' file not found". 
I google this issue but all the result is make .m file to .mm file. But It's just a h file...
I saw your pods also include this file and build success.

Is there something wrong with my environment configuration?

looking forward to your answer, thanks again:)
